src/common/darktable.h

```diff
@@ -303,7 +303,6 @@ typedef enum dt_debug_thread_t
 
 typedef struct dt_codepath_t
 {
-  unsigned int SSE2 : 1;
   unsigned int _no_intrinsics : 1;
```

Comment:
wouldn't this be better a gboolean?

Reply:
The struct contains bitfields of one bit each (only one after this PR).  `gboolean` may be incompatible, depending on its definition, since C bitfields are only allowed to be declared as `int` (optionally with signed or unsigned qualifier) or `_Bool`.